sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -126,7 +126,7 @@ class ServiceBusReceiver(
      The default value is 0, meaning messages will be received from the service and processed one at a time.
      In the case of prefetch_count being 0, `ServiceBusReceiver.receive` would try to cache `max_message_count`
      (if provided) within its request to the service.
```

Comment:
just checking that the changes in these file don't need to be made to async?

Reply:
No there was a spacing issue in the sync file that wasnt there in async. Thats why the change is only here 😊